app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for someone in people:
+        Person(someone["name"], someone["age"])
```

Comment:
1. Let's use list comprehension here
2. Let's save these person and return at the end instead of fetching them from Person.people.values()